wonjong/app.js

```diff
@@ -45,6 +45,60 @@ app.post("/user", async (req, res) => {
   res.status(201).json({ message: "userCreated" });
 });
 
+app.post("/post", async (req, res) => {
+  const { title, content, userID } = req.body;
+  await appDataSource.query(
+    `INSERT INTO posts(
+      title,
+      content,
+      user_id) VALUES (?,?,?)
+      `,
+    [title, content, userID]
+  );
+  res.status(201).json({ message: "postCreated" });
+});
+
+app.get("/post", async (req, res) => {
+  await appDataSource.query(
+    `SELECT 
+    u.id as userID,
+    u.profile_image as userProfileImage,
+    p.id as postingId,
+    p.title as postingTitle,
+    p.content as postingContent 
+    FROM users u
+    JOIN posts p ON u.id = p.user_id`,
+    (err, rows) => {
+      res.status(200).json(rows);
+    }
+  );
+});
+
+app.get("/user", async (req, res) => {
```

Comment:
```suggestion
app.get("/users/posts", async (req, res) => {
```
유저가 작성한 포스트 목록을 가져오는 api이므로 end-point에서도 이러한 계층을 `/`를 통해 나타내주면 조금 더 목적과 동작에 대한 직관적인 end-point를 작성할 수 있을 것 같습니다.